packages/definitions-parser/src/lib/definition-parser.ts

```diff
@@ -254,7 +254,7 @@ export function getFiles(
   moduleResolutionHost: ts.ModuleResolutionHost
 ): readonly FilesForSingleTypeScriptVersion[] {
   const errors = [];
-  const rootDir = dt.subDir("types").subDir(typingsData.typesDirectoryName);
+  const rootDir = dt.subDir("types").subDir(typingsData.subDirectoryPath);
```

Comment:
this is the version that includes /v15, eg `react/v15`, right? `rootDir` sounds like a misleading name for it.